### PR TITLE
Share Button to Project 

### LINKED ISSUE #1821: Share Button to Project
**Describe the new addon**

This addon would add a share button on project pages that would allow users to easily share projects on sites such as Twitter or Facebook via the built-in intent prompts. (Could be merged with the copy link button)

image

### PR DESCRIPTION
**Pull request by Genius398**
**Resolves**

Resolves #1821

**Changes**
I have added a button which when clicked takes the user to a WhatsApp Webpage with a pre-written message for sharing quickly.

Also, forgive me if there are some errors, cause this is the first time I'm contributing. It's also the first time I'm writing JSON (I stick to JS ☹)

### CODE REVIEW

File: addon.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    "name": "Share Button",
+    "description": "Provides a button with when clicked allows you to choose from a few websites and share a pre-framed message of your Scratch Project. Will be updated soon, but it can currently only supprot whatsapp",
+    "userstyles": [
+      {
+        "url": "button.css",
+        "matches": ["https://scratch.mit.edu/projects/*"]
+      }
+    ]
+,
+      "persistent_scripts": ["background.js"],
```
Comment (mxmou) [inline addon.json:11]: This needs to be a userscript because it interacts with the page.

**Review by mxmou — CHANGES_REQUESTED** [review]

File: addon.json (hunk 1 above)
Comment (Genius398) [reply addon.json:11]: What do you mean? Please explain

**Review by RedGuy12 — CHANGES_REQUESTED** [review]
go to https://github.com/Genius398/ScratchAddons/actions and enable it so your code can be automatically formatted.

**Comment by GrahamSH-LLK** [comment]
This is useless. People are going to write their own messages anyway.

**Pull request closed without merging**